pkg/testworkflows/testworkflowexecutor/executor.go

```diff
@@ -467,7 +468,23 @@ func (e *executor) initialize(ctx context.Context, workflow *testworkflowsv1.Tes
 		RunningContext:            request.RunningContext,
 	}
 
-	// Try to resolve tags initially
+	// Store the configuration if it is small and not sensitive
+	if request.Config != nil && testworkflows.CountMapBytes(request.Config) < 10*1024*1024 {
```

Comment:
```suggestion
	if testworkflows.CountMapBytes(request.Config) < 10*1024*1024 {
```

Nullish `request.Config` is still a config - it's an empty map that may still match.

Reply:
Also, worth to move the size as constant